src/utils/addresses.ts

```diff
@@ -0,0 +1,52 @@
+import { getAddress } from '@ethersproject/address'
+
+// returns the checksummed address if the address is valid, otherwise returns false
+export function isAddress(value: any): string | false {
+  try {
+    // Alphabetical letters must be made lowercase for getAddress to work.
+    // See documentation here: https://docs.ethers.io/v5/api/utils/address/
+    return getAddress(value.toLowerCase())
+  } catch {
+    return false
+  }
+}
+
+// Shortens an Ethereum address
+export function shortenAddress(address: string): string {
```

Comment:
was it intentional to remove the ability to set the number of chars at the beginning/end? I noticed `src/nft/components/details/DetailsContainer.tsx` for example used to only have two characters in the beginning but would receive 4 now.

Reply:
This PR vs Staging
![Screen Shot 2023-06-15 at 14 26 57 ](https://github.com/Uniswap/interface/assets/11512321/782b1e2e-0477-4429-9c2e-f5b8246aa25f)
![Screen Shot 2023-06-15 at 14 27 12 ](https://github.com/Uniswap/interface/assets/11512321/2c9bdb43-bf40-4abe-a4d1-1b3f605c240d)

Reply:
good catch!